src/test/java/net/minecraftforge/debug/ModNameTooltip.java

```diff
@@ -0,0 +1,48 @@
+package net.minecraftforge.debug;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.text.TextFormatting;
+import net.minecraftforge.event.entity.player.ItemTooltipEvent;
+import net.minecraftforge.fml.common.Loader;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.ModContainer;
+import net.minecraftforge.fml.common.eventhandler.EventPriority;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import net.minecraftforge.fml.relauncher.Side;
+
+@Mod(modid = "forgemodnametooltip", name = "ForgeModNameTooltip", version = "1.0", clientSideOnly = true)
+@Mod.EventBusSubscriber(Side.CLIENT)
```

Comment:
I'm not completely sure, but is giving a side to `@EventBusSubsciber` necessary when `@Mod.clientSideOnly` prevents the class from ever being loaded?